docs/components/renderer.rst

```diff
@@ -6,5 +6,51 @@
 Renderer
 ========
 
-..  warning::
-    This needs to be documented
+A Renderer transforms the :abbreviation:`AST (abstract syntax tree)` into a
```

Comment:
```suggestion
A Renderer transforms the :abbr:`AST (abstract syntax tree)` into a
```

At least in Sphinx it is abbr.

But I would use just `Abstract Syntax Tree (AST)` when mentioning first, so no need to hover it.

Reply:
The two textroles are synonyms